plugins/orchestrator/src/components/next/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface WorkflowOption {
```

Comment:
Can this name be improved? `WorkflowOption` doesn't tell much IMO 🤔

Reply:
Totally agree :-) I piggybacked on the BE naming.

Changing to `WorkflowSuggestion`